### DISCUSSION TITLE
cross referencing list

### DISCUSSION THREAD

**Discussion by syveqc** [dotnet/docfx#7994] [Q&A]
Hi, I want to crossreference to a method that takes a list as an argument, lets say it is
`MyNamespace.MyClass.MyMethod(List<MyOtherClass> arg)`.
Now I want to crossreference it as mentioned in <https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#advanced-more-options-for-cross-reference>. The link unfortunately does only explain the generic case, I have tried stuff like
`@"MyNamespace.MyClass.MyMethod(System.Collections.Generic.List<MyNamespace.MyOtherClass>)"`
or 
``@"MyNamespace.MyClass.MyMethod(System.Collections.Generic.List`MyNamespace.MyOtherClass)"``
but nothing seems to work. How can I reference this method?

**Comment by a-michelis** [comment]
Has your method a UID? -afaik, methods are embeded to the class files and do not have a UID on their own, they're but a section within the class file, so, you'd need to do something like.

@"MyNamespace.MyClass#MyMethod_anchor_or_smt_idunno"

I haven't fully understood how xrefs work, so take my words with a grain of salt

**Comment by syveqc** [comment]
Thanks for your answer! It does not have a UID right now, but googling how to set a UID got me to this website, <https://dotnet.github.io/docfx/spec/metadata_dotnet_spec.html>, which specifies that {} is used for this kind of reference, so
`@"MyNamespace.MyClass.MyMethod(System.Collections.Generic.List{MyNamespace.MyOtherClass})"`
finally worked :)
Thank you!

**Reply by a-michelis** [reply]
Oh, yes, the syntax used by `<see>` tags on .NET's XML documentation! I was thinking of mentioning it but i thought that it was a roslyn-only thing and i couldn't quite understand where your xref existed.

Glad it worked!
